#!/usr/bin/env python
PACKAGE = "ros_sees"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("aps_enabled",   bool_t,   0, "Enable APS Frame Capture",  False)
gen.add("dvs_enabled",   bool_t,   0, "Enable DVS Capture",  True)
gen.add("imu_enabled",   bool_t,   0, "Enable IMU Capture",  True)
gen.add("exposure", int_t, 0, "APS Frame Exposure Time [us]", 6000, 0, 30000);
gen.add("frame_rate", int_t, 0, "APS Frame Delay Time [us]", 1000, 0, 30000);
gen.add("gyro_scale", int_t, 0, "IMU Gyro Scale [0-3]", 1, 0, 3);
gen.add("accelerometer_scale", int_t, 0, "IMU Accelerometer Scale [0-3]", 1, 0, 3);


exit(gen.generate(PACKAGE, "ros_sees", "ros_sees"))
